src/main/scala/io/sdkman/changelogs/WebtauMigrations.scala

```diff
@@ -0,0 +1,35 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "044")
+class WebtauMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_webtau_candidate",
+    author = "MykolaGolubyev"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "webtau",
+      name = "webtau",
+      description =
+        "webtau - web test automation. test across REST-API/Graph QL/Browser/Database/CLI/Business-Logic",
```

Comment:
This is the description that appears on our website, and also when viewing the `sdk list` command. Can you please make this a little more descriptive and let it be a real sentence/paragraphl?

Reply:
I copied it as is from the webtau repo. Do you have any suggestions how can I improve it? 
Tool/API allows to write tests that touch one or multiple layers within one test or suite; generate rich report and other goodies. 
What is the size limit?

Reply:
There is no real size limit, but it is probably best to look at what other vendors are doing for general length and tone.

Reply:
updated description